### DISCUSSION TITLE
Poor contrast between "All" and "Unread" toggle state of Inbox

### DISCUSSION THREAD

**Discussion by gjsjohnmurray** [community/community#55919] [Accessibility]
### Select Topic Area

Product Feedback

### Body

![image](https://github.com/community/community/assets/6726799/d8a126a0-5981-4545-b6a0-1de509bcd352)

I find it hard to see at a glance which mode I am in. yes, the filter field to the right of the button-pair has text to tell me, but I feel it would be helpful for the selected button to contrast more with the unselected one.

**Comment by cehfisher** [comment]
Hello @gjsjohnmurray - thank you for providing this feedback! We will file this with our Accessibility team so that it can be reviewed and will follow-up here with any additional questions or updates. Thanks again!

**Comment by cehfisher** [comment]
Hello @gjsjohnmurray - could you please retest this again? There have been some component updates that should resolve this issue for you.

**Reply by gjsjohnmurray** [reply]
Much better. Thanks!
